spec/support/recurring_contributions_helper.rb

```diff
@@ -21,6 +21,7 @@ def build_transaction_mock(subscription_id = nil, id: 10)
                           status: 'waiting_payment',
                           amount: '31000',
                           payment_method: 'credit_card',
+                          boleto_url: 'https://pagar.me',
```

Comment:
O problema de usar o mock, é que a gente precisa passar isso na chamada dele :x 
Mas **se não** for muito custoso, seria interessante vc passar isso como params. Pq ai vc setaria isso na hora do setup do teste.
Na primeira vez que eu li o teste, eu não sabia de onde tava vindo essa url.